features/nanostack/mbed-mesh-api/mbed-mesh-api/WisunInterface.h

```diff
@@ -185,24 +185,40 @@ class WisunInterface : public MeshInterfaceNanostack {
      *
-     * Function stores new parameters to mbed-mesh-api and uses them when connect() is called next time.
-     * If device is already connected to the Wi-SUN network then device will restart network discovery after
-     * changing the network size.
+     * Function stores network size parameter to the mbed-mesh-api and uses it when connect() is called for the next
+     * time. If a device is already connected to the Wi-SUN network, then the device will restart network discovery
+     * after changing the network size.
+     *
+     * It is recommended to set the correct network size because the Wi-SUN network configuration parameters are
+     * adjusted based on the selected network size. A network configured for a small amount of devices may not work
+     * optimally for large number of devices. This is because the network bandwidth is divided with all the devices in
+     * the network. Enough bandwidth must be reserved for application data usage as well as the Wi-SUN network
+     * operations. Application should adapt to the network characteristics by using the InternetSocket methods
+     * get_stagger_estimate_to_address and get_rtt_estimate_to_address.
+     *
+     * The network size is measured as hundreds of devices that are expected to join to the network. For example,
+     * for a 400-device network set network size to 4. The Wi-SUN stack will internally use one of the four possible
+     * configuration buckets based on the selected network size and data rate. The configuration buckets are defined
+     * for a small, medium, large and xlarge networks.
+     *
+     * For example, with 150kbs data rate set network size to:
+     * - 1 to use small configuration bucket.
+     * - 8 to use medium configuration bucket.
+     * - 15 to use large configuration bucket.
+     * - 25 to use xlarge configuration bucket.
+     *
+     * By default Wi-SUN stack is configured to use medium sized network.
      *
-     * Default value: medium
-     * Small network size: less than hundred devices
-     * Medium network size: hundreds of devices
-     * Large network size: thousands of devices
-     * Certificate: used on testing
+     * The network size should be set to 0 when running certification tests.
      *
-     * When network size is changed, it will override all or some of the following configuration values:
+     * When the network size is changed, it will override all or some of the following configuration values:
      * - Timing settings set by set_timing_parameters() of the Wi-SUN interface.
      * - RPL settings set by rpl_parameters_set() of the Border Router interface.
      *
-     * When network size is changed, and if timing or RPL values should be other than defaults set by stack for the network size,
-     * they need to set again using above function calls.
+     * When the network size is changed, and if timing or RPL values should be other than defaults set by stack for the
```

Comment:
This is only change Border router RPL configs.

Reply:
There will be "configuration buckets based on the selected network size and data rate" for the four options, but is there also suitable configuration buckets for other values like"4"? or application need to handle other configurations by itself? 
In addition, there are two API list here, set_timing_parameters/rpl_parameters_set, is it meant once change network to "4", only there two parameters need to change? how?

Reply:
Only configuration Application needs to set is the network size and the stack will select the correct configuration that it will use based on the network size and data rate

All values from 1 to 254 will result in some selection made by stack so any value can be given even 4 meaning expected amount of devices in network is 400 devices.